runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropPlugin.java

```diff
@@ -99,19 +99,20 @@ public void onWidgetHidden(WidgetHiddenChanged event)
 			{
 				case MELEE:
 					if (spriteIDs.anyMatch(id ->
-						id == SpriteID.SKILL_ATTACK || id == SpriteID.SKILL_STRENGTH || id == SpriteID.SKILL_DEFENCE))
+                            id == SpriteID.SKILL_ATTACK || id == SpriteID.SKILL_STRENGTH || id == SpriteID.SKILL_DEFENCE
+                                    || id == SpriteID.SKILL_HITPOINTS))
```

Comment:
The indenting here seems to have blown up, probably because you've used spaces instead of tabs. Fix that and your Travis build will be successful.

Reply:
Thanks, hope my latest commit fixes this.

Reply:
Thanks for your patience with me :laughing:. I've got the CheckStyle plugin to work now, it shouldn't happen again from now on.